### PR TITLE
Added development section to README

### PR DESCRIPTION
**Pull request by seans3**
Added some notes on:

1) How to resolve dependencies after cloning repo
2) How to run unit tests on repo
3) How to develop in Kubernetes using this repo
4) How to run unit tests for this repo code within Kubernetes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,53 @@ definitions.
  - The spec generator that is responsible for dynamically generate 
 the final OpenAPI spec using web service routes or combining other 
 OpenAPI/Json specs.
+
+## Development
+
+After cloning this repo, within the top-level directory do the following to
+resolve dependencies:
```
Comment (apelisse) [inline README.md]: resolve and download?

File: README.md [hunk 2]
```diff
@@ -12,3 +12,53 @@ definitions.
...
+
+```
+$ go test ./pkg/...
+```
+
+### Testing with Kubernetes
+
+Assuming you already have a `k8s.io/kubernetes` repo, and `k8s.io/kube-openapi` repo:
+
+* Remove vendored `kube-openapi` within `k8s.io/kubernetes`. From top-level directory:
```
Comment (lavalamp) [inline README.md]: I would prefer to make this not necessary than document it :)

Comment (mbohlool) [reply README.md]: Agreed. lets remove it from documentation and add an issue in the repo to fix it.

**Pull request closed without merging**